README.md

```diff
@@ -139,113 +137,9 @@ __Objective-C__
 }];
 ```
 
-#### Available tracking events model classes
-* VSDKAddToCart
-* VSDKPageView
-* VSDKProductClick
-* VSDKProductCustomization
-* VSDKProductFeedback
-* VSDKProductImpression
-* VSDKProductView
-* VSDKProductViewDetails
-* VSDKPurchase
-* VSDKRefund
-* VSDKRemoveFromCart
-* VSDKSearch
-* VSDKSubscription
-
-#### Create your custom tracking event
-If none of the available tracking events matches your needs you can also extend `TrackingEvent` and create your own. 
-
-__Beware!__ _Custom tracking events might not be interpreted by our services and will have limited functionality (logging and basic statistics). Please try to use one of the existing tracking events model classes or at least inherit from one of those classes when creating a custom event to ensure you make the most out of Velocidi's CDP._
-
-##### Objective-C
-
-Create a new Cocoa Touch class that inherits from `VSDKTrackingEvent` (or any of the other tracking event model classes).
-
-Example:
-
-__CustomEvent.h__
-```objectivec
-#import <VelocidiSDK/VelocidiSDK.h>
-
-@interface CustomEvent : VSDKTrackingEvent
-
-// Place below the desired fields you want to add to the tracking event
-@property (nullable) NSString *customField;
-
-@end
-```
-
-__CustomEvent.m__
-```objectivec
-#import "CustomEvent.h"
-
-@implementation CustomEvent
-
-- (instancetype) init {
-    if(self = [super init]){
-        self.type = @"customEvent"; // Only necessary when inheriting directly from VSDKTrackingEvent
-    }
-    return self;
-}
-
-@end
-```
-
-This new class can then be imported and used like any other tracking event:
-
-```objectivec
-#import "CustomEvent.h";
-
-...
-
-CustomEvent * trackingEvent =  [[CustomEvent alloc] init];
-trackingEvent.clientId = @"RandomSiteId";
-trackingEvent.siteId = @"RandomClientId";
-trackingEvent.customField = @"RandomCustomField";
-
-[VSDKVelocidi.sharedInstance track: trackingEvent] 
-```
-
-##### Swift
-Due to limitations of the framework we use to serialize classes to JSON ([JSONModel](https://github.com/jsonmodel/jsonmodel)), a Swift class that inherits from `VSDKTrackingEvent` or any tracking event model class won't have its properties serialized when sending the event. To have this functionality you will have to create your custom event in Objective-C and import it into Swift:
+A list of the available tracking event classes can be found [here](https://ios.developers.velocidi.com/tracking-model-classes-list.html). If none of them fit the desired action, you can also create your own [custom tracking event](https://ios.developers.velocidi.com/custom-tracking-events.html)
```

Comment:
```suggestion
A list of the available tracking event classes can be found [on the iOS developers website](https://ios.developers.velocidi.com/tracking-model-classes-list.html). If none of them fit the desired action, you can also create your own [custom tracking event](https://ios.developers.velocidi.com/custom-tracking-events.html).
```

Reply:
Don't like the wording you suggested. "_the iOS developers website_" looks you're talking about Apple's Official website on iOS development 😅 I'd leave it as it is.

Reply:
I tend to not endorse links behind the word _here_. Also, it is not a very good practice: http://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/

Reply:
Good point! Changed in 05c9e35